models/Movie.models.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+
+const movieSchema = new mongoose.Schema({
+  title: String,
+  director: String,
+  stars: Array,
+  image: String,
+  description: String,
+  showtimes: Array,
```

Comment:
We can be a little more specific doing this 
stars:[String]
showtimes : [String]